src/main/java/uk/gov/hmcts/reform/laubackend/idam/repository/UserDeletionAuditFindRepository.java

```diff
@@ -60,11 +63,13 @@ public Page<UserDeletionAudit> findUserDeletion(
 
         final List<String> queryParts = new LinkedList<>();
         queryParts.add(SELECT);
+        queryParts.add(WHERE);
         queryParts.add(TIME_RANGE_CRITERIA);
 
         Map<String, String> usedParams = addSearchCriteria(queryParts, params);
         queryParts.add(ORDER);
-        
+        queryParts.add(DESC);
```

Comment:
yes Before the change in findUserDeletion query it was DESC so for that query it will remain same.   for findAllDeletedUsers if sort is provided it will sort according to ASC or DESC otherwise it will  be DESC.